lib/tasks/prune_reports.rake

```diff
@@ -50,15 +91,19 @@ UNITS:
       exit 1
     end
 
+    if condition != "!="
+      condition = conditions[condition]
+    end
+
```

Comment:
Remove the `if`, since above you'll set the default to `not` which means you can unconditionally access the hash.